draft-ietf-tls-tlsflags.xml

```diff
@@ -150,6 +150,13 @@
             <t> A server MAY indicate support for flag-type features in a flags extension of NewSessionTicket (NST).
                 This message has no client-side response.</t>
             <t> In summary, unsolicited flags may appear only in ClientHello, CertificateRequest, and NewSessionTicket.</t>
+            <t> An implementation that receives an invalid tls_flags extension MUST terminate the TLS handshake with a
+                fatal illegal_parameter alert. Such invalid tls_flags extensions include: <list style="symbols">
+                <t> A zero-length extension</t>
+                <t> Multiple tls_flags extensions for the same message </t>
```

Comment:
RFC8446 already prohibits this on the sender. We don't do a good job of defining the rules for the receiver, but that should probably be done uniformly across all extensions, rather than just in this one.